include/picongpu/fields/FieldTmp.tpp

```diff
@@ -198,6 +198,26 @@ namespace picongpu
         PMACC_KERNEL(Kernel{})(mapper.getGridDim(), numWorkers)(mapper, thisBox, modifyingBox);
     }
 
+    template<uint32_t AREA, bool sumSqrt>
+    float_X FieldTmp::calculateSum()
```

Comment:
This can be removed by [local reduce](https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/include/pmacc/device/Reduce.hpp)